Sources/ApodiniObserve/ConfiguredLogger.swift

```diff
@@ -0,0 +1,232 @@
+    public init(id: UUID = UUID(), label: String? = nil, logLevel: Logger.Level? = nil) {
+        self.init(id: id, logLevel: logLevel, label: label)
+    }
+}
+
+extension ConfiguredLogger {
+    private var endpointMetadata: Logger.Metadata {
+        [
+            "name": .string(self.blackboardMetadata.endpointName),
+            "parameters": .array(self.blackboardMetadata.endpointParameters.map { parameter in
+                    .string(parameter.description)
+            }),
+            "operation": .string(self.blackboardMetadata.operation.description),
+            "endpointPath": .string(self.blackboardMetadata.endpointPathComponents.value.reduce(into: "", { partialResult, endpointPath in
+                partialResult.append(contentsOf: endpointPath.description)
+            })),
+            "version": .string(self.blackboardMetadata.context.get(valueFor: APIVersionContextKey.self)?.debugDescription ?? "unknown"),
+            "handlerType": .string(String(describing: self.blackboardMetadata.anyEndpointSource.handlerType)),
+            "handlerReturnType": .string(String(describing: self.blackboardMetadata.handleReturnType.type)),
+            "serviceType": .string(self.blackboardMetadata.serviceType.rawValue)
+        ]
+    }
+    
+    private var exporterMetadata: Logger.Metadata {
+        [
+            "type": .string(String(describing: exporterTypeMetadata.exporterType)),
+            "parameterNamespace": .array(exporterTypeMetadata.parameterNamespace.map { .string($0.description) })
+        ]
+    }
+    
+    private func getInformationMetadata(from informationSet: InformationSet) -> Logger.Metadata {
+        informationSet.reduce(into: [:]) { partialResult, info in
+            if let anyHTTPInformation = info as? AnyHTTPInformation {
+                if let auth = anyHTTPInformation.typed(Authorization.self) {
+                    partialResult[Authorization.header] = .string(auth.type)
+                } else if let cookies = anyHTTPInformation.typed(Cookies.self) {
+                    partialResult[Cookies.header] = .dictionary(
+                        cookies.value.reduce(into: [:]) { partialResult, cookie in
+                            partialResult[cookie.key] = .string(cookie.value)
+                        }
+                    )
+                } else if let etag = anyHTTPInformation.typed(ETag.self) {
+                    partialResult[ETag.header] = .string(etag.rawValue)
+                } else if let expires = anyHTTPInformation.typed(Expires.self) {
+                    partialResult[Expires.header] = .string(expires.rawValue)
+                } else if let redirectTo = anyHTTPInformation.typed(RedirectTo.self) {
+                    partialResult[RedirectTo.header] = .string(redirectTo.rawValue)
+                } else {
+                    partialResult[anyHTTPInformation.key.key] = .string(anyHTTPInformation.value)
+                }
+            }
+        }
+    }
```

Comment:
I don't really like, that we have an explicit dependency on `ApodiniHTTPProtocol`. As far as I can see, this is the only reason why this dependency exists?

First of all, is there any value in saving e.g. the Cookie Information as a `.dictionary` and don't just use the `.string` raw value format? Afaik this is the only reason currently for enumerating every currently supported HTTPInformation type (Etag, Redirect, etc). I mean, any other header is currently also just passed as the rawValue representation, and any future HTTPInformation is probably treated the same unless someone **manually** adjust this implementation right here.
So for one, I would propose to not look at the possible `InformationInstantiables` of an Information type but instead always just use the value/rawValue of the "base" `Information` type. In the example of `AnyHTTPInformation` one would just use the string header and the string value. Specifically, as any third party can introduce their own typed `HTTPInformation` implementation, I don't think it make sense to try to maintain an enumeration of those to somehow provide more "detailed" Metadata?

Secondly, this currently only covers `AnyHTTPInformation` Information and ignores any other potential Information instances. I think we should be able to use the `InformationClass` feature to provide a more generic implementation for this (refer to e.g. `HTTPHeaderInformationClass` for a fully working example).
Would something like the following work:
1. Create a `StringKeyedStringInformationClass` e.g. 
```swift
public protocol StringKeyedStringInformationClass: InformationClass {
    var entry: (key: String, value: String) { get }
}
```
Maybe you could realize some kind of `StringKeyDecodableInformationClass` reusing your `IntermediateRepresentation`. You just need to keep it type erased as we later use the information class type to filter for those in an array loop (refer to step 3 or e.g. see how it is done with the `HTTPHeaderInformationClass` in the `HTTPHeaders+InformationKey.swift` file).
The `StringKeyedStringInformationClass` could maybe be stilled introduced to be a more specific version of a `StringKeyDecodableInformationClass`(?).
2. Ideally, the above introduced InformationClass is placed inside Apodini as a "default" InformationClass. Such that the `ApodiniHTTPProtocol` package can use it without requiring a dependence between it and `ApodiniAuthorize`.
3. And finally, `ApodiniObserve` would use a logic like the following to collect all the information (replacing the code above):
```swift
 informationSet.reduce(into: [:]) { partialResult, info in
     if let info = info as? StringKeyedStringInformationClass {
         partialResult[info.entry.key] = .string(info.entry.value)
     }
 }
```


Would that make sense to you? Could that even work?